spec/spec_helper.rb

```diff
@@ -36,6 +36,7 @@
   end
 
   config.filter_run_excluding appveyor_agents: true unless ENV['APPVEYOR_AGENTS']
+  config.filter_run_excluding windows: true unless ENV['BOLT_WINDOWS']
```

Comment:
We generally do this via tags in the Rakefile, not separate environment variables.

Reply:
Actually not sure why this is necessary at all.

Reply:
Ah. I guess all the other tests run from macOS or Linux. Definitely a non-Windows view of development.